action/flow/definition/definition.go

```diff
@@ -294,6 +294,9 @@ const (
 
 	// LtError denotes an error link
 	LtError LinkType = 3
+
+	//LinkType denotes an complex linker expression
+	LtFunction  LinkType = 4
```

Comment:
What is a "complex linker expression" ?  Is this just the link expression that WI used?  Did they ever use LtExpression?    

It seems like this is just an LtExpression, that uses another "language".  @mellismi Maybe flows should say what expression contrib/language the flow is using?  I don't think its a good idea for a process to use some expression lang for one link and then another expression lang for another.  I say if the new stuff is a superset of what we have now, lets use that as LtExpression, and make all "expressions" a contribution and we have a default implementation.  

@lixingwang we currently support simple conditionals...   $flow.myval < 10, $activity[a1].out == "blah", etc.  Basically our current one follows goland conditional operators.

Reply:
@fm-tibco Yes. this one we need to be discussed. I tried to integration into LtExpression but we are using different language.

For the complex linker expression which more resolve static function/expr mapper linker.  such as: `string.contains("hello","he")`, we can use function to linker mapper whenever the function returns a boolean.

Reply:
That is fine if it a superset of what we have.  We just need the simple stuff to work, it’s ok if it can do more.  @mellismi thoughts?

Reply:
agree, its ok to be a superset but it needs to be the same basic language structure

Reply:
Luckily we only have simple go comparison operators at the moment, if the new stuff has the same, I say we replace whats there... and afterwards come up with a contribution for expressions.  In case someone wants to plugin a light weigh mapper/expression or even precompiled mappings/expressions.

Reply:
@fm-tibco Right.  For WI case we have our own contributed linker mapper. we dropped what OSS have today.   but since we move to OSS now I use new type LtFunction.   

Come up with a contribution with a different type same the with mapper is a good idea :)